docs/src/components/ComponentDoc/ComponentProps/ComponentPropsOutline.tsx

```diff
@@ -0,0 +1,28 @@
+import * as _ from 'lodash'
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+import { link } from 'docs/src/utils/helpers'
+
+const ComponentPropsOutline: any = ({ displayNames, parentDisplayName }) => {
+  if (displayNames.length < 2) return null
+
+  const items: string[] = _.map(displayNames, displayName =>
+    displayName === parentDisplayName
+      ? displayName
+      : displayName.replace(parentDisplayName, `${parentDisplayName}`),
+  )
```

Comment:
What it should do? I don't see any difference in links:

![image](https://user-images.githubusercontent.com/14183168/65320048-57287e80-dba1-11e9-8a5c-b1b2a93c077c.png)

Reply:
It's a cruft. Removed.